zenhappens/src/services.html

```diff
@@ -0,0 +1,371 @@
+            <td class="table__price">$40</td> 
+        </tr>
+        <tr>
+            <td>Brazilian</td>
+            <td class="table__price">$60</td> 
+        </tr>
+        <tr>
+            <td>Stomach strip</td>
+            <td class="table__price">$8</td> 
+        </tr>
+        <tr>
+            <td>Inner thigh</td>
+            <td class="table__price">$16</td> 
+        </tr>
+        </table>
+        </div>
+
+
+         <div class="price-list--flex">
+        <table class="table-style table-style__left">
+        <tr>
+            <td>Back</td>
+            <td class="table__price">$35</td>
+        </tr>
+        <tr>
+            <td>Lower/Upper Back</td>
+            <td class="table__price">$40</td> 
+        </tr>
+        <tr>
+            <td class="table__add-detail">Underarm</td>
+            <td class="table__price">$60</td> 
+        </tr>
+        <tr>
+            <td>Shoulder</td>
+            <td class="table__price">$8</td> 
+        </tr>
+        <tr>
+            <td>Chest</td>
+            <td class="table__price">$16</td> 
+        </tr>
+        <tr>
+            <td>Stomach</td>
+            <td class="table__price">$28</td> 
+        </tr>
+         <tr>
+            <td>Chest/Stomach</td>
+            <td class="table__price">$58</td> 
+        </tr>
+        </table>
+
+        <table class="table-style">
+           <tr>
+            <td>Leg (Half)</td>
+            <td class="table__price">$40</td>
+        </tr>
+        <tr>
+            <td>Leg (Full)</td>
+            <td class="table__price">$68</td> 
+        </tr>
+        <tr>
+            <td>Arm (Half)</td>
+            <td class="table__price">$36</td> 
+        </tr>
+        <tr>
+            <td>Arm (Full)</td>
+            <td class="table__price">$60</td> 
+        </tr>
+        </table>
+        </div>
+        </section>
+        </section>
+
+    
+    <section class="container">
+        <h2 class="price-list__title">Dry Cupping</h2>
+        <div class="price-list--flex">
+        <table class="table-style table-style__left">
+        <tr>
+            <td rowspan="2" class="u-sansserif">
+            <input type="checkbox" class="read-more-state" id="cupping"/>
+            <p class="read-more-wrap">Cupping is a type of alternative therapy that originated in China. It involves placing cups on the skin to create suction. The suction facilitates healing with blood flow, as well as the flow of “qi” in the body.
+            <span class="read-more-target">Qi is a Chinese word meaning life force. Cupping increases blood circulation to the area where the cups are placed. This can relieve muscle tension, which can improve overall blood flow and promote cell repair. It can also help form new connective tissues and create new blood vessels in the tissue. People use cupping to complement their care for a host of ailments and conditions.</span></p>
+            <label for="cupping" class="read-more-trigger"></label>
```

Comment:
```suggestion
            <label for="cupping" class="read-more-trigger" aria-hidden="true"></label>
```
Since the entire text is experienced by a screenreader, I would recommend hiding the label from screen readers.